3.4/administration-cluster.md

```diff
@@ -208,3 +208,57 @@ To distribute shards onto the new _DBServer_ either click on the
 `Distribute Shards` button at the bottom of the `Shards` page in every
 database.
 
+The clean out process can be monitored using the following script,
+which periodically prints the amount of shards that still need to be moved.
+It is basically a countdown to when the process finishes.
+The current status is logged every 5 seconds. You may adjust it by
+changing the number on the line `internal.wait(5);`.
```

Comment:
Note to self: change this sentence here and update the script (sleep duration as second parameter).